packages/axios-extension/src/abap/lrep-service.ts

```diff
@@ -256,6 +270,36 @@ export class LayeredRepositoryService extends Axios implements Service {
         }
     }
 
+    /**
+     * Get system info.
+     *
+     * @param language
+     * @param cloudPackage name
+     * @returns the system info object
+     */
+    public async getSystemInfo(language: string = 'EN', cloudPackage?: string): Promise<SystemInfo> {
+        try {
+            const params = {
+                'sap-languag': language
+            };
+            if (cloudPackage) {
+                params['package'] = cloudPackage;
+            }
+
+            const response = await this.get(`${DTA_PATH_SUFFIX}system_info`, { params });
+            this.tryLogResponse(response, 'Successful getting system info.');
+            return JSON.parse(response.data) as SystemInfo;
+        } catch (error) {
+            this.log.error('Getting system data failed.');
+            this.log.debug(error);
+            if (isAxiosError(error) && error.response?.status === 405) {
```

Comment:
that one is interesting. for what is a newer version required? Is the service not available in this version or ... ?

Reply:
Fixed